amps/ags/rm-community/rm-community-repo/config/alfresco/module/org_alfresco_module_rm/messages/action-service.properties

```diff
@@ -45,4 +45,4 @@ rm.action.create.transfer.container.child-error-message=You can't create items i
 rm.action.create.transfer.child-error-message=You can't create items in Transfer Folders.
 rm.action.create.record.folder.child-error-message=You can only create records in record folders and this was a {0}.
 rm.action.transfer-non-editable=You can't edit transfer folder or container metadata.
-
+rm.action.node.frozen.error-message=unable to perform action {0} because node is frozen or has frozen children
```

Comment:
All the messages are starting with a capital letter.  When adding new strings it is good to review them with someone from docs team, we used to work with @oliveralfresco2018

Reply:
Also the other messages seem to end in a full stop.  Definitely worth getting a message that's approved by UA (e.g. Oliver).